### PR TITLE
Add missing examples for 'removed' functionality

### PR DESCRIPTION
**Pull request by mhaskel**
We've removed some of the nice wrappers, but everything that was possible should still be possible.

### CODE REVIEW

File: examples/backports.pp [hunk 1]
```diff
@@ -1 +1,32 @@
-# TODO
+$location = $::apt::distcodename ? {
+  'squeeze' => 'http://backports.debian.org/debian-backports',
+  'wheezy'  => 'http://ftp.debian.org/debian/',
+  default   => 'http://us.archive.ubuntu.com/ubuntu',
```
Comment (daenney) [inline examples/backports.pp]: Use `archive.ubuntu.com`, without the `us.` prefix. That should give people a geographically more desirable mirror.

File: examples/backports.pp [hunk 2]
```diff
@@ -1 +1,32 @@
...
+  $release = $::apt::distcodename
+} else {
+  $repos   = 'main universe multiverse restricted'
+  $key     = '630239CC130E1A7FD81A27B140976EAF437D05B5'
+  $release = $::apt::distcodename
+}
+
+# set up the actual backports
+apt::pin { 'backports':
+  before   => Apt::Source['backports'],
```
Comment (daenney) [inline examples/backports.pp]: This is not needed. `pin` uses `setting` which by default will notify `apt::update`. The order here isn't really important. Doesn't really hurt either though.

File: examples/disable_keys.pp [hunk 3]
```diff
@@ -0,0 +1,5 @@
+#Note: This is generally a bad idea. You should not disable keys.
```
Comment (daenney) [inline examples/disable_keys.pp]: Not disabling keys, disabling verifying repository signatures / allowing installation of packages from unsigned / unverified sources.

File: examples/backports.pp (hunk 2 above)
Comment (daenney) [inline examples/backports.pp]: I'm almost starting to consider pulling backports back in, just not providing a pin facility and letting people always manage that separately. What do you think?

Comment (daenney) [inline examples/backports.pp]: As an aside, pin's are immediately reflected in apt, they do not require an `apt-get update` so we should alter `apt::pin` to not notify `apt-get update` as to not needlessly refresh things on a run.

Comment (mhaskel) [inline examples/backports.pp]: erm, I mean, I think the reason we pulled this out in the first place is it's really just wrapping an `apt::source` and an `apt::pin`. The logic above that is just figuring things out based on your env. Idunno. No strong feelings either way :neutral_face:

**Comment by daenney** [comment]
```
examples/backports.pp - WARNING: indentation of => is not properly aligned on line 28
examples/backports.pp - WARNING: indentation of => is not properly aligned on line 29
```

**Comment by mhaskel** [comment]
womp. fixed.

**Pull request merged**